.circleci/config.yml

```diff
@@ -14,46 +15,45 @@ commands:
             echo "export RUST_BACKTRACE=1" >> $BASH_ENV
             echo "export CARGO_INCREMENTAL=0" >> $BASH_ENV
 
-  test-start:
+  setup-uv:
     steps:
-      - checkout
       - run:
-          name: environment
+          name: Installing uv
+          command: |
+            UV_SHA256=f6788fd45a25ea1033a671eaa9ece6c9b004ce5c9cf09552effadde567a7d7a5
+            UV=uv-x86_64-unknown-linux-gnu
+            UV_VERSION=0.6.13
```

Comment:
What's the process for upgrading the version of `uv` we use on CI?

Reply:
uuh download both, then update here ... yeah, I should document.